libmavconn/cmake/Modules/CheckGeographicLibDatasets.cmake

```diff
@@ -0,0 +1,27 @@
+##
+# This module verifies the installation of the GeographicLib datasets and warns
+# if it doesn't detect them.
+##
+
+find_path(GEOGRAPHICLIB_GEOID_PATH NAMES geoids PATH_SUFFIXES share/GeographicLib share/geographiclib)
+find_path(GEOGRAPHICLIB_GRAVITY_PATH_ NAMES gravity PATH_SUFFIXES share/GeographicLib)
+find_path(GEOGRAPHICLIB_MAGNETIC_PATH_ NAMES magnetic PATH_SUFFIXES share/GeographicLib)
+
+if(NOT GEOGRAPHICLIB_GEOID_PATH)
+  message(WARNING "No geoid model datasets found. This will result on a SIGINT! Please execute the script install_geographiclib_dataset.sh in /mavros/scripts")
+else()
+  message(STATUS "Geoid model datasets found in: " ${GEOGRAPHICLIB_GEOID_PATH}/geoid)
+  set(GEOGRAPHICLIB_GEOID_PATH ${GEOGRAPHICLIB_GEOID_PATH}/geoid)
+endif()
+if(NOT GEOGRAPHICLIB_GRAVITY_PATH_)
+  message(WARNING "No gravity field model datasets found. Please execute the script install_geographiclib_dataset.sh in /mavros/scripts")
+else()
+  message(STATUS "Gravity Field model datasets found in: " ${GEOGRAPHICLIB_GRAVITY_PATH_}/gravity)
+  set(GEOGRAPHICLIB_GRAVITY_PATH ${GEOGRAPHICLIB_GRAVITY_PATH_}/gravity)
+endif()
+if(NOT GEOGRAPHICLIB_MAGNETIC_PATH_)
+  message(WARNING "No magnetic field model datasets found. Please execute the script install_geographiclib_dataset.sh in /mavros/scripts")
+else()
+  message(STATUS "Magnetic Field model datasets found in: " ${GEOGRAPHICLIB_MAGNETIC_PATH_}/magnetic)
+  set(GEOGRAPHICLIB_MAGNETIC_PATH ${GEOGRAPHICLIB_MAGNETIC_PATH_}/magnetic)
+endif()
```

Comment:
Wrong? Wrong why? Check the all PR and you will see that's also done. This is a check during build time that warns people that they basically did not fully followed the installation instructions. A SIGINT is launched when that happens. What I can also do is verbose the SIGINT with a info msg so that people run the install script. Besides that, this is perfectly acceptable IMO and doesn't harm anyone.